types/archiver/index.d.ts

```diff
@@ -1,51 +1,65 @@
 // Type definitions for archiver 1.3
 // Project: https://github.com/archiverjs/node-archiver
-// Definitions by: Esri <https://github.com/archiverjs/node-archiver>
+// Definitions by: Dolan Miu <https://github.com/dolanmiu>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /* =================== USAGE ===================
 
     import Archiver = require('archiver);
     var archiver = Archiver.create('zip');
-    archiver.pipe(FS.createWriteStream('xxx'));
-    archiver.append(FS.createReadStream('xxx'));
+    archiver.pipe(fs.createWriteStream('xxx'));
+    archiver.append(fs.createReadStream('xxx'));
     archiver.finalize();
 
  =============================================== */
 
-/// <reference types="node" />
```

Comment:
Please leave this in, even if it's not technically necessary as the dependency is pulled in by `glob`.